gen/MotController_data.yml

```diff
@@ -120,19 +121,32 @@ c_MotController_Create1:
       # -> 0/1 refers to pidIdx
       'pid0_iaccum': 0.0,
       'pid0_error': 0,
+      'pid0_prevErr': 0,
       'pid0_errorDerivative': 0.0,
       'pid0_feedback': FeedbackDevice.None_,
+      'pid0_notFirst': False,
+      'pid0_outBeforRmp': 0,
+      'pid0_target': 0,
       
       'pid1_iaccum': 0.0,
       'pid1_error': 0,
+      'pid1_prevErr': 0,
       'pid1_errorDerivative': 0.0,
       'pid1_feedback': FeedbackDevice.None_,
+      'pid1_notFirst': False,
+      'pid1_outBeforRmp': 0,
+      'pid1_target': 0,
       
       # slotIdx
       'profile_slot_select': 0,
       # pidIdx
       'pid_slot_select': 0,
       
+      # motion magic parameters
+      'cruise_velocity': 0,
```

Comment:
This all looks fantastic, but I'm going to bikeshed here... shouldn't cruise_velocity and acceleration have magic or mm or something to indicate that they're part of motion magic?